gems/administrate/CVE-2016-3098.yml

```diff
@@ -0,0 +1,14 @@
+---
+gem: administrate
+cve: 2016-3098
+title: Cross-site request forgery (CSRF) vulnerability in administrate gem
+date: 2016-04-01
+url: https://cve.mitre.org/cgi-bin/cvename.cgi?name=2016-3098
```

Comment:
Ah, looking for more of an advisory here. Why not http://seclists.org/oss-sec/2016/q2/0 or something like that?